index.html

```diff
@@ -0,0 +1,119 @@
+    <p class="location">@ Warsaw, Poland</p>
+  </section>
+  <div id="mainProgram" class="moveUp"></div>
+  <main class="dflex-column py-5">
+    <h2 class="white-title">Main program</h2>
+    <div class="bar"></div>
+    <section class="program">
+      <article>
+        <img src="img\G key.png" alt="pic">
+        <h3 class="program-titles">Opening</h3>
+        <p>Opening of the event with the presentation of the performers Lorem ipsum 
+          dolor sit amet consectetur adipisicing.
+        </p>
+      </article>
+      <article>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">Nocture op</h3>
+        <p>A devastating melody that lures you in at pianissimo, 
+          and one most likely recognised for its use in 
+          Roman Polanski’s The Pianist.
+        </p>
+      </article>
+      <article>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">Fantaisie-Impromptu</h3>
+        <p>The posthumously published Fantaisie-Impromptu is one 
+          of Chopin’s most ostentatiously improvised works, and a 
+          fantastic show piece for the piano.
+        </p>
+      </article>
+      <article>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">24 Preludes</h3>
+        <p>Each prelude contains a unique idea or feeling, from 
+          the pianistic precipitation in ‘Raindrop’ to the haunting 
+          E minor prelude, which was played at Chopin’s own funeral.
+        </p>
+      </article>
+      <article>
+        <img src="img\F key.png" alt="pic">
+        <h3 class="program-titles">Closing</h3>
+        <p>Closing and presentation of the band, Lorem ipsum dolor 
+          sit amet consectetur adipisicing elit. Ut, blanditiis!
+        </p>
+      </article>
+    </section>
+    <div id="ticket" class="moveUp"></div>
+    <a href="#">
+      <button type="button" class="join mobile-mode">Get a Ticket</button>
+    </a>
+  </main>
+  <div id="performerSection" class="moveUp"></div>
+  <section class="performers-and-pieces">
+    <h2 class="performers-and-pieces-title">Performers</h2>
+    <div class="bar"></div>
+    <ul class="performers gap-5"></ul>
+    <button type="button" class="d-flex justify-content-center align-items-center gap-2 more" id="more">
+      <p class="m-0">MORE</p>
+      <i class="arrow down" id="arrow"></i>
+    </button>
+  </section>
+  <section class="desktop-mode partner dflex-column" id="partnerSection">
+    <h2 class="performers-and-pieces-title" id="partner">Partner</h2>
+    <div class="bar"></div>
+    <ul class="d-flex">
+      <li>Kultura Dostepna</li>
+      <li>TFCI</li>
+      <li>Google</li>
+      <li>STEANWAY LYNGDORF</li>
+    </ul>
+  </section>
+  <footer class="desktop-mode dflex-column flex-row align-items-center justify-content-center p-3 gap-3">
+    <img src="img\logo.png" alt="logo" width="250px">
+    <p>1830 Concert Chopin <br> Some Right Reserved.</p>
+  </footer>
```

Comment:
- [ ] as the previous reviewer asked,  I can't see the footer in the mobile version please make sure to add it there as well in order to match the Design 
<img width="346" alt="Screenshot 2022-11-24 at 17 46 06" src="https://user-images.githubusercontent.com/61790587/203834183-333b447f-7ff6-43af-96a5-ac4e84f8c98f.png">
<img width="772" alt="Screenshot 2022-11-24 at 17 48 06" src="https://user-images.githubusercontent.com/61790587/203834497-8f167fb5-86e3-4a95-baf9-c99997c0cc93.png">
- Design 
<img width="475" alt="Screenshot 2022-11-24 at 17 49 39" src="https://user-images.githubusercontent.com/61790587/203834614-0ee5f18e-8c2d-44e1-95ba-ba7431f3045a.png">